Readme.md

```diff
@@ -1,24 +1,20 @@
-# WebMemex/memextension
+# WorldBrain's WebMemex
```

Comment:
Not sure you understood: I tried to point out that the changes in your WorldBrain fork ended up in your pull request to the WebMemex repo. It's often easiest to make a new branch for each pull request you make.

As for how you name your fork, that's another question.